src/NuGet.Core/NuGet.Packaging/Core/FrameworkReference.cs

```diff
@@ -4,13 +4,12 @@
 using System;
 using System.Collections.Generic;
 using NuGet.Shared;
+using static NuGet.Common.ComparisonUtility;
```

Comment:
I think [this](https://github.com/NuGet/NuGet.Client/pull/2829/files#diff-1bf3c9468063aed0d10fb059f72efc43R172) is easier to understand than `using static`; you can't easily know where `FrameworkReferenceNameComparer` comes from without an IDE or searching.

Reply:
Not sure what you are proposing. 

```cs
using NuGet.Common; 

//
    return ComparisonUtility.FrameworkReferenceNameComparer.Equals(Name, other.Name);
//
```

?

Reply:
Yes, exactly!